helm/rbac-app/values.yaml

```diff
@@ -0,0 +1,31 @@
+# TEMPLATE-APP: This is set as a reasonable default, feel free to change.
+
+name: rbac
+namespace: kube-system
+serviceType: managed
+
+project:
+  branch: "[[ .Branch ]]"
+  commit: "[[ .SHA ]]"
+
+image:
```

Comment:
I would remove these unused values and add a values.schema.json https://intranet.giantswarm.io/docs/organizational-structure/teams/cabbage/app-updates/helm-values-schema/

Reply:
This is coming from template app, but I guess here makes no sense

Reply:
Also the binding will be commented, it is an example for now